### DISCUSSION TITLE
Disabling acceleration

### DISCUSSION THREAD

**Discussion by jsejcksn** [pilotmoon/Scroll-Reverser#111] [Ideas]
Hello @invariant

I just finished reading your most recent blog post, [Notes on Scroll Reverser 1.8; and my thoughts on the future of the app](https://pilotmoon.com/blog/2020/12/09/scroll-reverser-1-8).

During your hiatus from updating Scroll Reverser, another developer has published [UnnaturalScrollWheels](https://github.com/ther0n/UnnaturalScrollWheels). The features related to customizing acceleration behavior seem like a very natural fit for Scroll Reverser—namely:

> - [x] Disable scroll acceleration
>   - `n` Number of lines to scroll per wheel click
> - [x] Disable mouse acceleration _(1:1 mouse/cursor movement)_

What are your thoughts on incorporating this into Scroll Reverser? Do you intend to implement these features?

---

Unrelated: Do you accept donations via any other method than PayPal?

**Comment by pilotmoon** [comment]
Hi and thanks for your engagement! I'll have to look into the detail, but I'm definitely going to add a discrete scroll option which may be a similar thing as number of lines per scroll wheel click (similar to another tiny app called DiscreteScroll). Not sure about mouse acceleration - it might be out of scope.
I only have PayPal set up I'm afraid.

**Comment by pilotmoon** [comment]
Related issue: #65

**Comment by pilotmoon** [comment]
Closing this discussion now, as the feature was already added in July 2021 with version 1.8.1.
